src/js/game/LevelMVC/AssetLoader.js

```diff
@@ -105,6 +105,11 @@ module.exports = class AssetLoader {
         pngPath: `${this.assetRoot}images/Leaves_Spruce_Snowy_Decay.png`,
         jsonPath: `${this.assetRoot}images/Leaves_Spruce_Snowy_Decay.json`
       },
+      Nether_Portal: {
+        type: 'atlasJSON',
+        pngPath: `${this.assetRoot}images/Nether_Portal.png`,
+        jsonPath: `${this.assetRoot}images/Nether_Portal.json`
```

Comment:
Hmm, why are we adding the sprites to `Blocks.png` and also adding a `Nether_Portal.png`?

Reply:
Ah, missed that when taking out Nether_Portal.png

The phaser.sprite.animation frame stuff was much more ready to just pull from blocks.png (like it does for lava and water) than it was for adding a new file destination, for whatever reason. That can be taken out 👍